DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/PolylineToHostConverter.cs

```diff
@@ -0,0 +1,63 @@
+﻿using ArcGIS.Core.Geometry;
+using ArcGIS.Core.Internal.Geometry;
+using ArcGIS.Desktop.Mapping;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.ArcGIS3.Geometry;
+
+[NameAndRankValue(nameof(SOG.Polyline), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
+public class PolylineToHostConverter : ISpeckleObjectToHostConversion, IRawConversion<SOG.Polyline, Polyline>
+{
+  private readonly IConversionContextStack<Map, Unit> _contextStack;
+
+  public PolylineToHostConverter(IConversionContextStack<Map, Unit> contextStack)
+  {
+    _contextStack = contextStack;
+  }
+
+  public object Convert(Base target) => RawConvert((SOG.Polyline)target);
+
+  public Polyline RawConvert(SOG.Polyline target)
+  {
+    double f = Units.GetConversionFactor(target.units, _contextStack.Current.SpeckleUnits);
+    List<double> coordinates = target.value;
+
+    // POC: retrieve here exact wkt. Not hard coded!
```

Comment:
thank you for POC commenting this, my mind was a bit kabooom when I saw the long string! :D

Reply:
thanks to Oguzhan:)